tickplate.js

```diff
@@ -1,15 +1,21 @@
 'use strict';
+const vm = require('vm');
+const fs = require('fs');
+const path = require('path');
 
-const tickplate = (strings, ...keys) => {
-  console.dir({ strings, keys });
-  return (values) => {
-    console.dir({ values });
-    const result = [strings[0]];
-    keys.forEach((key, i) => {
-      result.push(values[key], strings[i + 1]);
-    });
-    return result.join('');
-  };
+const tickplate = (template, sandbox, callback) => {
```

Comment:
It's not a good design choice to force `template` to be a filename. The template might come from a database, or it can be generated dynamically, or your code can be run in an environment where there's no filesystem (at least, exposed to your code). If you absolutely sure that templates stored in files should be ready to use without writing three extra lines of code to do `fs.readFile`, add a separate convenience function to render a template from file, but the template engine itself must know nothing about the origin of a string which a template is.